src/main/java/nerdhub/textilelib/mixins/MixinPortalBlock.java

```diff
@@ -13,14 +13,14 @@
 @Mixin(PortalBlock.class)
 public abstract class MixinPortalBlock {
 
+    //FIXME this does technically work, but fires way too often
     //TODO map this to attemptPortalSpawn
```

Comment:
Yeah lets get rid of the `//TODO map this to attemptPortalSpawn` as i know the forgename and so asie will most likely not accept the mapped name

Reply:
done